server/heroku/logs.go

```diff
@@ -24,9 +25,10 @@ func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter,
 	}
 
 	var form PostLogsForm
-
 	if err := Decode(r, &form); err != nil {
-		return err
+		if err.Error() != "EOF" {
+			return fmt.Errorf("error decoding request: %v", err)
```

Comment:
i think then we're also continuing confusion around: https://github.com/remind101/empire/pull/867#discussion_r65992662, ie. hoping for underlying annotation